internal/ted448/scalar.go

```diff
@@ -199,5 +219,15 @@ func (z *Scalar) Mul(x, y *Scalar) {
 	z64.toScalar(z)
 }
 
-// IsZero returns true if z=0.
-func (z *Scalar) IsZero() bool { z.Red(); return *z == Scalar{} }
+// Inv calculates z = 1/x mod order.
+func (z *Scalar) Inv(x *Scalar) {
```

Comment:
Should include a big warning that this is not constant time.  Why not use an addition chain?  (Or have not got around to that yet?)

Reply:
Added an exponentiation method.